src/Microsoft.TestPlatform.CoreUtilities/Constants.cs

```diff
@@ -22,5 +22,11 @@ public class Constants
         /// Datacollector process name, without file extension(.exe/.dll)
         /// </summary>
         public const string DatacollectorProcessName = "datacollector";
+
+        /// <summary>
+        /// Number of character should be logged on child process exited with
+        /// error message on stadard error.
+        /// </summary>
+        public const int StandardErrorMaxLength = 20480; // 20 KB
```

Comment:
I don't see a reason for increasing it by 5 times. 4096 was a reasonable buffer, we can probably make ti 8192 (2x), if user wants more info, they can always go ahead check the logs.

Reply:
Made it 8192